lua/autorun/server/sv_chip_lister.lua

```diff
@@ -115,9 +115,11 @@ local function isPlateVisible( ent, ply )
 end
 
 -- Can a player see at least one chip lister?
+-- Give listers as false to only check for the HUD setting (i.e. no lister entities exist).
 local function canSeeALister( ply, listers )
     if not IsValid( ply ) then return false end
     if ply:GetInfoNum( "cfc_chiplister_hud_persist", 0 ) == 1 then return true end
+    if not listers then return end
```

Comment:
What's the point of this? you should be returning earlier here, no need to loop over the listUsers only to return empty each time

Reply:
When no lister ents exist, I still need to figure out who does/doesn't have the HUD enabled, to figure out:
A) Does at least one person have the HUD enabled, i.e. do we bother running an update at all?
B) Who do we need to network the data to?